src/setupWorker/glossary.ts

```diff
@@ -65,6 +66,12 @@ export type StartOptions = SharedOptions & {
    * of all registered Service Workers on the page.
    */
   findWorker?: FindWorker
+
+  /**
+   * Disable the keep alive or use a custom interval.
+   * Defaults to 10000 (ms)
+   */
+  keepAlive?: false | number
```

Comment:
I take a general rule of thumb to parametrize public stuff. Since this is rather internal interval, I think we should skip the ability to change it. That way we shift the responsibility of it having optimal value to ourselves, rather than the end user.

Reply:
In this case, I've just removed the option altogether. It's safe just to keep it a 5s and hide the behavior, as it's what users expect.